google/cloud/forseti/common/gcp_api/securitycenter.py

```diff
@@ -174,3 +181,53 @@ def create_finding(self, finding, organization_id=None, source_id=None,
                 finding.get('properties').get('violation_data')
                 .get('full_name'))
             raise api_errors.ApiExecutionError(full_name, e)
+
+    def list_findings(self, source_id):
+        """Lists all the findings in CSCC.
+
+          Args:
+              source_id (str): Unique ID assigned by CSCC, to the organization
+                  that the violations are originating from.
+
+          Returns:
+              object: An API response containing all the CSCC findings.
+        """
+        response = self.repository.findings.list(parent=source_id)
+        return response
+
+    def update_finding(self, finding, finding_id, state, event_time,
+                       source_id=None):
+        """Updates a finding in CSCC.
+
+        Args:
+            finding (dict): Forseti violation in CSCC format.
+            name (str): Name of the CSCC finding.
+            state (str): State of the CSCC finding.
+            event_time (str): Event time of the the CSCC finding.
+            source_id (str): Unique ID assigned by CSCC, to the organization
+                that the violations are originating from.
+            finding_id (str): id hash of the CSCC finding.
+
+        Returns:
+            dict: An API response containing one page of results.
+        """
+        # alpha api
+        if not source_id:
+            return
+
+        # beta api
+        try:
+            LOGGER.debug('Updated finding with beta api.')
```

Comment:
It would be good to add the finding id here to the log.

LOGGER.debug('Updated finding: %s: ', finding_id)